setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+version = "0.2.0"
+
+def _read_long_desc():
+    with open("README.md") as fp:
+        return fp.read()
+
+setup(
+  name="tern",
+  version=version,
+  author="VMWare Inc",
+  author_email="FIXME@FIXWHAT.THEEMAIL",
+  url="https://github.com/vmware/tern/",
+  description=("An inspection tool to find the metadata of the packages" 
+    " installed in a container image"),
+  long_description=_read_long_desc()
+  license="BSD-2.0",
+  keywords="Distribution, Container, Cloud-Native",
+  classifiers = [
+    'Development Status :: 4 - Beta',
```

Comment:
Should be `3 - Alpha` and hopefully not Beta before this merges :)

Reply:
Ah! I'll update this :+1: